packages/toolkit/config/webpack/plugins.js

```diff
@@ -103,9 +103,19 @@ module.exports = ({
 					return removeDistFolder(style);
 				}
 
-				const isBlockAsset = useBlockAssets
-					? buildFiles[options.chunk.name].match(/\/blocks\//)
-					: options.chunk.name.match(/-block$/);
+				const fullPath = options.chunk.entryModule.identifier().split('!').pop();
```

Comment:
I think this has the potential to cause issues... and the reason is that things can get really confusing.....

If `useBlocksAsset` is being used then `filenames` is overridden [here](https://github.com/10up/10up-toolkit/blob/develop/packages/toolkit/config/webpack/entry.js#L18) and that makes things work mostly fine.

But when `useBlocksAsset` is not set to true which is the default behavior then `filenames.blockCSS` is `blocks/[name]/editor.css`. Then here's what happens:

- The CSS is generated at `blocks/[name]/editor.css`
- Any CSS file imported from the JS of the block would also be generated at `blocks/[name]/editor.css` (currently this ends up being generated at `css/[name].css` - which is the value of `filenames.css`.
 
 I actually think this is the issue @roseg43 is running into https://github.com/10up/10up-toolkit/pull/267#issuecomment-1410799411 (@roseg43  are you using `useBlocksAssets`?)

I think the following code would be backwards compatible:

```js
// this returns what you want but would be undefined for CSS 
// that comes from non-css files (e.g CSS imported via JS)
const fullPath = options.chunk.entryModule.resource; 
if (!fullPath) {
 return filenames.css;
}
```

I did just a few tests but I'm thinking we should document the current behavior just to be extra sure of how it works and make sure the changes here do not break the current behavior.

Reply:
The local theme we use for testing things has `useBlocksAssets` enabled so that's prob why @Antonio-Laguna wasn't running into this issue.

Reply:
🙇 thanks a lot @nicholasio I would have sworn that I had tested this in both scenarios but you're more than right! I've updated this

Reply:
Ah yep, I definitely didn't have it enabled!